### ISSUE TITLE
Hide compounds, make compound_rank_score exclusive?

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#4105]
Currently, the hide compounds rank score is used inclusively, so that e.g. if the user enters hide 10, variants up to and including 10 are hidden. This is intuitive on one level, but the user often operates with the level of last included rank score in mind, so having to think "I want to keep 10, so enter 9 there". Not huge, and everyone is different but lets at least have a quick discussion.. 

![Screenshot 2023-10-02 at 12 37 30](https://github.com/Clinical-Genomics/scout/assets/758570/6411b899-3dad-4b09-b655-7bc111bc1048)

**Comment by northwestwitch** [comment]
Perhaps modify the label  "compound tank score"  into "compound rank score >" ?

**Comment by dnil** [comment]
Absolutely, we should update the label if we change it - though I would prefer "compound rank score <", as the feature is to hide variants. And no tanks! 😸

**Comment by northwestwitch** [comment]
Then it has to be <= no?

**Comment by dnil** [comment]
Well, depending on if we agree to change it or not, and given that we have trouble discussing it, we should probably add a few more words in the tooltip to make sure it is clear. It does give feedback through what is hidden/dismiss-colored so it's not that it is super confusing, or that it can't be understood if one looks carefully. More that one can read it differently on quick use, and that one probably goes to the dialog with the lowest number to keep in mind rather than highest to hide.
